### PR TITLE
Clean up product v2 js structure

### PR DESCRIPTION
**Pull request by zuk3975**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 8.1.x
| Description?      | Product v2 js structure was very confusing/unclear, and we can fix it now or never (because once 8.1 is shipped every change will be required to be defined as bc break) :smile: . So I moved some services around and IMO its now much cleaner, all the services related to specific subdomain are close to each other instead of being spread around.
| Type?             | refacto
| Category?         | BO
| BC breaks?        | no (techincally there are lots of bc breaks here, but everything is only related to product page v2)
| Deprecations?     | no
| How to test?      | UI tests should be enough (no new visual features, but whole product v2 page js was affected), usually when something wrong is imported build should fail.
| Fixed ticket?     | 
| Related PRs       | 
| Sponsor company   |

### CODE REVIEW

**Comment by kpodemski** [comment]
In my opinion, it should be merged before the beta release, so that people can get familiar with the final structure. What do you think @jolelievre ?

**Review by kpodemski — DISMISSED** [review]

File: admin-dev/themes/new-theme/js/pages/product/category/categories-service.ts [hunk 1]
```diff
@@ -26,6 +26,7 @@
 /* eslint-disable max-len */
```
Comment (jolelievre) [inline admin-dev/themes/new-theme/js/pages/product/category/categories-service.ts]: This file should be in the services folder, which should rather be named `services` than `service` by the way IMHO 🤔

**Review by jolelievre — APPROVED** [review]
Thanks @zuk3975

File: admin-dev/themes/new-theme/js/pages/product/category/categories-service.ts (hunk 1 above)
Comment (jolelievre) [reply admin-dev/themes/new-theme/js/pages/product/category/categories-service.ts]: Old comment, you can ignore it

**Comment by jolelievre** [comment]
Runnin campaign https://github.com/jolelievre/ga.tests.ui.pr/actions/runs/4142350250

**Comment by zuk3975** [comment]
> Runnin campaign https://github.com/jolelievre/ga.tests.ui.pr/actions/runs/4142350250

BO orders failing and something in FO, doesn't seem to be related

**Comment by jolelievre** [comment]
I reran the failed runs, personally I always see Orders 01 fail on the first time and it works on second run I don't know the reason

**Review by kpodemski — APPROVED** [review]

**Comment by zuk3975** [comment]
ping @PrestaShop/qa-team automated tests passes except one in FO. The one seems to be unrelated, because PR only changes admin new theme js files, can you approve it so it can be merged?

**Comment by HanaRebaiQA** [comment]
Hello @zuk3975 

Could you please add the `Waiting for QA` label and we will check it by launching the automatic tests?

thanks!

**Comment by kpodemski** [comment]
@HanaRebaiQA 

Automatic tests:
https://github.com/jolelievre/ga.tests.ui.pr/actions/runs/4142350250

No need to run them again, I guess.

**Comment by HanaRebaiQA** [comment]
Hello @kpodemski 

Thank You! 
For the fail in FO, I checked on the nightly of 8.1.x and it also exists https://github.com/PrestaShop/PrestaShop/actions/runs/4159259352/jobs/7195153341#step:6:1516

@zuk3975 So, it's QA :heavy_check_mark: 

Thanks!

**Comment by nicosomb** [comment]
Thank you @zuk3975 and @HanaRebaiQA !

**Pull request merged**